stablehlo/dialect/AssemblyFormat.h

```diff
@@ -198,6 +209,24 @@ void printCustomCallTarget(AsmPrinter& p, Operation*, StringAttr target);
 
 ParseResult parseCustomCallTarget(AsmParser& parser, StringAttr& target);
 
+// HloDim - Print an ArrayRef of integrer values with brackets.
```

Comment:
Proposed rewording:

```
IntArray - Print an array of ints with brackets.
Unlike DimensionSizes, doesn't have special handling for kDynamicSize.
```

Feedback:
  1. I think that using "dim" in this name is confusing with respect to `DimensionSizes`. Let's just say `IntArray`?
  2. "array of ints" to be consistent with above.

Reply:
As discussed in separate thread, would like this to use DimensionSizes, but need to make sure that is always used as dimension sizes first. Will look into that in followup PR.

Reply:
Accepted proposed rewording.